### PR TITLE
Finalize 5.1 release notes

### PR DESCRIPTION
**Pull request by jdmgoogle**
This will close out #281 and signal the official 5.1 release.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -6,15 +6,27 @@ This document includes release notes for the [VIP Spec][vip] for versions
...
   * Issue [#298](https://github.com/votinginfoproject/vip-specification/issues/298)
+  * Use `xs:IDREFS` instead of `xs:IDREF maxOccurs="unbounded"`
+    * This was also causing some issues with Java code generation
+  * Move Move `SequenceOrder` from `Candidate` to `BallotSelectionBase`
+  * Update `CandidateContest` to support multiple `PrimaryPartyIds`
+  * Add `Gender` to `Person`
+  * Make `TermType` in `Term` optional
+  * Add "borough" to `DistrictType`
+* Support for city-level wildcarding
+  * Issue [#299](https://github.com/votinginfoproject/vip-specification/issues/299)
```
Comment (jktomer) [inline HISTORY.md]: There doesn't seem to be an issue #299?

**Comment by jktomer** [comment]
LGTM modulo nit above

**Comment by jdmgoogle** [comment]
Typo. Was issue #289. Fixed.

**Comment by jdmgoogle** [comment]
Unless there are any objections raised before noon ET today, I'm going to merge this in and then do the official release of VIP 5.1.

**Comment by jungshadow** [comment]
Here is a gif titled "force-merge." Seems appropriate ;)
Bulls mascot running into someone holding drinks

Awesome work, everyone :+1:

**Pull request merged**